test-engineering/contract-tests/volumes/client/scenarios.yml

```diff
@@ -408,3 +408,120 @@ scenarios:
+                image_url: 'https://example.com/us_wa_desktop_windows01.jpg'
+                image_size: null
+                impression_url: 'https://example.com/us_wa_desktop_windows?id=0001'
+                url: 'https://www.example.com/us_wa_desktop_windows'
+              - id: 66789
+                name: 'Example ORG'
+                click_url: 'https://example.org/us_wa_desktop_windows?version=16.0.0&key=7.2&ci=8.9&ctag=E1DE38C8972D0281F5556659A'
+                image_url: 'https://example.org/us_wa_desktop_windows02.jpg'
+                image_size: null
+                impression_url: 'https://example.org/us_wa_desktop_windows?id=0002'
+                url: 'https://www.example.org/us_wa_desktop_windows'
+      - request:
+          service: contile
+          method: GET
+          path: '/v1/tiles'
+          headers:
+            # Contile maps the User-Agent Header value to os-family and form-factor parameters
+            # The following value will result in os-family: windows and form-factor: desktop
+            - name: User-Agent
+              value: 'Mozilla/5.0 (Windows NT 10.0; rv:10.0) Gecko/20100101 Firefox/91.0'
+            # Contile looks up the IP address from this header value and maps it to proxy information.
+            # We use a random IP address from the range specified by the CIDR network notation "216.160.83.56/29"
+            # from https://github.com/maxmind/MaxMind-DB/blob/main/source-data/GeoLite2-City-Test.json
+            # The following value will result in country-code: US and region-code: WA
+            - name: X-Forwarded-For
+              value: '216.160.83.62'
+        response:
+          status_code: 200
+          content:
+            tiles:
+              - id: 22345
+                name: 'Example COM'
+                click_url: 'https://example.com/us_wa_desktop_windows?version=16.0.0&key=22.1&ci=6.2&ctag=1612376952400200000'
+                image_url: 'https://example.com/us_wa_desktop_windows01.jpg'
+                image_size: null
+                impression_url: 'https://example.com/us_wa_desktop_windows?id=0001'
+                url: 'https://www.example.com/us_wa_desktop_windows'
+              - id: 66789
+                name: 'Example ORG'
+                click_url: 'https://example.org/us_wa_desktop_windows?version=16.0.0&key=7.2&ci=8.9&ctag=E1DE38C8972D0281F5556659A'
+                image_url: 'https://example.org/us_wa_desktop_windows02.jpg'
+                image_size: null
+                impression_url: 'https://example.org/us_wa_desktop_windows?id=0002'
+                url: 'https://www.example.org/us_wa_desktop_windows'
+      - request:
+          service: partner
+          method: GET
+          path: '/records/'
+          headers:
+            - name: 'accept'
+              value: '*/*'
+        response:
+          status_code: 200
+          content:
+            records:
+              - count: 2
```

Comment:
The sequence diagram for `success_tiles_cache_expired` shows 1 `GET` request from contile to the `/tilesp/desktop` partner endpoint, whereas the scenario is expecting 2 requests.

Reply:
Thanks for catching this! I should have put `success_tiles_cached_for_identical_proxy_params` (formally `success_tiles_cache_with_same_form_and_region`) as the title.